packages/@okta/vuepress-site/docs/guides/oin-lifecycle-mgmt-overview/index.md

```diff
@@ -5,35 +5,35 @@ meta:
     content: Learn how to build a lifecycle management app integration for the Okta Integration Network.
 ---
 
-Lifecycle management refers to the process of provisioning and deprovisioning application access as a user moves through an organization. Onboarding, role change, transfer, and offboarding employee processes require provisioning user accounts in workforce applications and enforcing corporate security policies in a timely manner.
+Lifecycle management refers to the process of provisioning and deprovisioning app access as a user moves through an org. Onboarding, role change, transfer, and offboarding employee processes require provisioning user accounts in workforce apps. It also requires enforcing corporate security policies in a timely manner.
```

Comment:
I believe we capitalize Workforce as it's an Okta thing.

```suggestion
Lifecycle management refers to the process of provisioning and deprovisioning app access as a user moves through an org. Onboarding, role change, transfer, and offboarding employee processes require provisioning user accounts in Okta Workforce apps. It also requires enforcing corporate security policies in a timely manner.
```

Reply:
Hmm - the fact that we read it two separate ways means it needs to be clarified. 👍 

(And the fact that Workday as an HR app is involved is also bringing additional confusion sauce to the table.)

I'd suggest changing it to just have "require provisioning user accounts in Okta and other third-party apps"?